src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.scss

```diff
@@ -14,6 +14,12 @@
     flex-direction: column;
   }
 
+  .interstitial-media-container {
+    display: flex;
```

Comment:
please look at the `.media-container` css to see how this should look like. This new container is now the parent for `.interstitial-media` and needs to behave the same as the old parent `.media-container`:
- display: flex
- flex-direction: column

Also, it should take the available space from its parent, so flex 1 and align-self: stretch need to be present.

The order should be: props related to the parent (flex, align-self) and props related to the children (display flex, flex-direction, position relative).